src/tests/test_solve_postfix_equation.py

```diff
@@ -67,6 +67,15 @@
     ("1235.563 _", 35.1505760977),
     ("59872369.547896589 _", 7737.7237963),
 
+    ("0 ^ 1", 0),
+    ("1 ^ 1", 1),
+    ("1 ^ 0", 0),
+    ("43 ^ 2", 1849),
+    ("100 ^ 3", 1000000),
+    ("-2 ^ -1", -0.5),
+    ("2 ^ 11", 2048),
+    ("1.678 ^ 5.963", 21.89951196284224),
+
```

Comment:
mocnina by měla být ve tvaru v postfixu "x(základ) y(exponent) ^", ty to máš infixově "x(základ) ^ y(exponent)"